src/frontend/src/styles/variables.scss

```diff
@@ -45,6 +45,7 @@ $Blue20: #D8EAFD;
 $Blue30: #C1DDFC;
 $Blue80: #3470B1;
 $Blue90: #1E5189;
+$Blue100: #2D5DD8;
```

Comment:
- $Blue100 is supposed to be #013366, and #2D5DD8 is not the correct hex value
- The correct blue is already defined as a variable in $Focus, please use that for this

Reply:
Updated the styling based on the comments.